test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -89,3 +90,13 @@ def test_pytorch_mpi(pytorch_training, ec2_connection, gpu_only, py3_only):
 @pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
 def test_nvapex(pytorch_training, ec2_connection, gpu_only):
     execute_ec2_training_test(ec2_connection, pytorch_training, PT_APEX_CMD)
+
+
+@pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
```

Comment:
Can u make it run on only one GPU instance like this(https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py#L83)